x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/collector/cluster/ClusterStatsMonitoringDocTests.java

```diff
@@ -331,6 +331,7 @@ public void testToXContent() throws IOException {
                                                                             needToEnableTLS);
 
         final BytesReference xContent = XContentHelper.toXContent(doc, XContentType.JSON, false);
+        logger.error(xContent.utf8ToString());
```

Comment:
can this be removed now?

Reply:
https://github.com/elastic/elasticsearch/pull/35625/commits/39392e114fbf16b730d47e98a6cba384082efaf9